mat/gsvd.go

```diff
@@ -382,15 +400,15 @@ func (gsvd *GSVD) VTo(dst *Dense) *Dense {
 		capCols: c,
 	}
 	dst.Copy(tmp)
-	return dst
 }
 
 // QTo extracts the matrix Q from the singular value decomposition, storing
 // the result in-place into dst. Q is size c×c.
-// If dst is nil, a new matrix is allocated. The resulting Q matrix is returned.
 //
-// QTo will panic if the receiver does not contain a successful factorization.
-func (gsvd *GSVD) QTo(dst *Dense) *Dense {
+// If dst is empty, QTo will resize dst to be c×c. If dst is
+// non-empty, QTo will panic if dst is not c×c. QTo will also
+// panic if the receiver does not contain a successful factorization.
```

Comment:
"When dst is non-empty, QTo will panic if dst is not c×c."

Reply:
Done.